package.sh

```diff
@@ -13,10 +13,11 @@ chmod +x docker-debify
   --depends tzdata \
 
 # Create possum rpm
+# The flag to avoid packaging build files that are not needed and cause conflict with conjur-ui on install
```

Comment:
```
The '--rpm-rpmbuild-define' flag is added to avoid packaging build 
files that are not needed and cause conflict with conjur-possum on install
```

Reply:
Changed